src/tools/common.jam

```diff
@@ -1029,6 +1033,19 @@ local rule address-model-tag ( name : type ? : property-set )
     return [ $(property-set).get <address-model> ] ;
 }
 
+# Create a tag for the architecture and model
+# <architecture>x86 <address-model>32 would generate "x32"
```

Comment:
So windows-on-intel would generate "x32" and "x64". What about ARM, PPC, etc. ? Do they generate "A32", "P64", etc. ? I'd rather Boost respects preexisting nomenclature for naming architectures, including "x86", "x86_64", "arm", "aarch64", etc.